siddhi-extension-archetype/siddhi-map/src/main/resources/archetype-resources/component/pom.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Missing license file

Reply:
According to the code review comment, all files under the archetype-resources folder don't need to put the license header. Because these files are a resource for generating the project using the archetype. Hence I didn't put the license header.